app/models/activity.rb

```diff
@@ -325,10 +327,18 @@ def is_project?
   end
 
   def is_gcrf_funded?
-    parent.present? && associated_fund.roda_identifier_fragment == "GCRF"
+    parent.present? && associated_fund.is_gcrf_fund?
   end
 
   def is_newton_funded?
-    parent.present? && associated_fund.roda_identifier_fragment == "NF"
+    parent.present? && associated_fund.is_newton_fund?
+  end
+
+  def is_newton_fund?
+    roda_identifier_fragment == "NF"
```

Comment:
@leeky  Could I check with you that this way of identifying the funds is OK long-term, and better than checking that the string is contained within the fragment?

Reply:
@CristinaRO Well, `roda_identifier` for a fund should never change (and we override `Activity#roda_identifier_compound=` to prevent it) as every child activity would also need to be updated if it did.